packages/iTwinUI-react/src/core/Toast/Toaster.tsx

```diff
@@ -115,8 +107,37 @@ export default class Toaster {
     this.updateView();
   }
 
+  private async createContainer() {
+    const container = getContainer(TOASTS_CONTAINER_ID) ?? getDocument()?.body;
+    if (!container) {
+      return;
+    }
+
+    const toastWrapper = <ToastWrapper ref={this.toastsRef} />;
+
+    try {
```

Comment:
Add a comment that this is for 17 and 18 support.

Reply:
added